spring/src/main/java/io/seata/spring/annotation/GlobalLock.java

```diff
@@ -27,30 +27,48 @@
  * declare the transaction only execute in single local RM,<br/>
  * but the transaction need to ensure records to update(or select for update) is not in global transaction middle
  * stage<br/>
- *
  * use this annotation instead of GlobalTransaction in the situation mentioned above will help performance.
  *
+ * @author skyesx
+ * @author WangLiang/王良
+ * @author selfishlover
+ * @author jimin
+ * @author linkedme@qq.com
  * @see io.seata.spring.annotation.GlobalTransactionScanner#wrapIfNecessary(Object, String, Object) // the scanner for TM, GlobalLock, and TCC mode
  * @see io.seata.spring.annotation.GlobalTransactionalInterceptor#handleGlobalLock(MethodInvocation) // the interceptor of GlobalLock
  * @see io.seata.spring.annotation.datasource.SeataAutoDataSourceProxyAdvice#invoke(MethodInvocation) // the interceptor of GlobalLockLogic and AT/XA mode
  */
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ElementType.METHOD,ElementType.TYPE})
+@Target({ElementType.METHOD, ElementType.TYPE})
 @Inherited
 public @interface GlobalLock {
     /**
      * customized global lock retry internal(unit: ms)
      * you may use this to override global config of "client.rm.lock.retryInterval"
      * note: 0 or negative number will take no effect(which mean fall back to global config)
-     * @return
+     * <br/>
+     * will be removed in v1.6, please use the new {@link GlobalLock#lockRetryInterval()}
+     *
+     * @return lock retry interval
      */
+    @Deprecated
     int lockRetryInternal() default 0;
 
```

Comment:
you can use spring @AliasFor to make the old lockRetryInternal and the new lockRetryInterval "merge" into one value, so as to keep compatibility and simply a lot of code(basically, translate those misspelled and no need to add more test).